src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+*Servlet class responsible for user registration.
```

Comment:
nit: increase the indent by 2 spaces as per this style guide: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Reply:
I tried to fix it and realized that I actually did double indentation in my original files (they looked just like the Google-provided files like LoginServlet.java,etc...)? I'm quite confused

Reply:
I think she's just referring to the actual comment, not the code. So like:
/**
 *Servlet class responsible for user registration.
 */
(Just match up the stars) I think the original code was indented just fine.

Reply:
My example didn't work...but yeah just match up each new line star with the first star in the first row.

Reply:
@krtejeda got it now, thanks!

Reply:
@krtejeda @ElleTojaroon fixed it again, let's do a final review so we can merge :)

Reply:
@ElleTojaroon can you update the change request? thanks!